aws/tf/modules/sra/databricks_workspace/workspace_security_modules/uc_catalog/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "aws_account_id" {
+  type = string
+}
+
+variable "resource_prefix" {
+  type = string
+}
+
+variable "databricks_account_id" {
+  type = string
+}
+
+variable "workspace_id" {
+  type = string
+}
+
+variable "uc_catalog_name" {
```

Comment:
This variable is used to name the UC workspace bucket.  Since the docs advise against using `.` in bucket names, maybe a validation or description calling out this restriction is warranted.

Reply:
this is generated by us `  uc_catalog_name         = "${var.resource_prefix}-catalog-${module.databricks_mws_workspace.workspace_id}"`

so shouldn't have to worry